waku/v2/waku_rln_relay/rln/wrappers.nim

```diff
@@ -51,30 +53,69 @@ proc membershipKeyGen*(ctxPtr: ptr RLN): RlnRelayResult[IdentityCredential] =
 
   return ok(identityCredential)
 
-proc createRLNInstanceLocal*(d: int = MerkleTreeDepth): RLNResult =
+type RlnTreeConfig* = ref object of RootObj
```

Comment:
Is this type needed to be public?
```suggestion
type RlnTreeConfig = ref object of RootObj
```

Reply:
Thanks, addressed in [f8774ee](https://github.com/waku-org/nwaku/pull/1782/commits/f8774ee0ec9a7d62bcef103c5b86d35b507cdb7d)